tests/UI/dashboard.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { expect, test } from '@playwright/test';
+import { BasePage } from "../../core/pageObject/Base";
+const logger = require('logger');
+
+let basePage;
+let dashboardPage;
+let loginPage;
+
+test.beforeEach(async ({ page }) => {
+    basePage = new BasePage(page);
+    dashboardPage = basePage.getPage('dashboard');
+    loginPage = basePage.getPage('login');
+    await basePage.goto('web');
+    await loginPage.loginAsTestUser();
+});
+
+test('Check that user can add new dashboard', async () => {
+    logger.info('Starting test: Check that user can add new dashboard');
+    await dashboardPage.addNewDashboard();
+    await dashboardPage.fillDashboardPopupInputsAndSave('New Dashboard1', 'Some Description2')
+    await expect.soft(dashboardPage.dashboardAddedSuccessfully).toBeVisible();
+    await expect.soft(dashboardPage.dashboardNameFromMenu).toHaveText('New Dashboard1');
+    logger.info('Test: Check that user can add new dashboard - Passed');
+});
+
+test('Check that user can delete newly add dashboard', async () => {
+    logger.info('Starting test: Check that user can delete newly add dashboard');
+    await dashboardPage.addNewDashboard();
+    await dashboardPage.fillDashboardPopupInputsAndSave('New Dashboard2', 'Some Description2')
```

Comment:
You might also add the Dashboard class and pass the dashboard object instead of passing plain text to PO methods.